eightpoints/guzzle-bundle/7.0/config/packages/eight_points_guzzle.yaml

```diff
@@ -1,17 +1,35 @@
+# EightPoints GuzzleBundle
+# A full list of configuration options can be found here:
+# https://github.com/8p/EightPointsGuzzleBundle/blob/master/src/Resources/doc/configuration-reference.md
 eight_points_guzzle:
+    # (de)activate logging/profiler; default: %kernel.debug%
+#    logging: true
+
     clients:
         my_client:
             # Write here the host where to do requests
-            # base_url: "http://target.url"
+#            base_url: "http://target.url"
```

Comment:
should probably be reverted

Reply:
the idea was to separate comments and uncommented properties so it's easier to find options than can be (de)activated/set.

Reply:
it's also standard behavior/position to comment/uncomment by using an IDE (at least for PhpStorm and Eclipse)